helm/istio/charts/tracing/templates/jaeger-all-in-one.yaml

```diff
@@ -0,0 +1,78 @@
+{{ if and (eq .Values.provider "jaeger") (eq .Values.jaeger.template "all-in-one") }}
+apiVersion: jaegertracing.io/v1
+kind: "Jaeger"
+metadata:
+  name: "jaeger"
+  namespace: {{ .Release.Namespace }}
+  labels:
+    chart: {{ template "tracing.chart" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+spec:
+  strategy: allInOne
+
+  allInOne:
+    image: {{ .Values.jaeger.hub }}/all-in-one:{{ .Values.jaeger.tag }}
+    options:
+      log-level: debug
+    annotations:
+      {{- range $key, $value := .Values.jaeger.annotations }}
+        {{ $key }}: {{ $value | quote }}
+      {{- end }}
+
+  query:
```

Comment:
The `query` and `collector` nodes are not actually relevant for the `allInOne` strategy - I think I may have misinformed about that. So instead, the `options/query/base-path` should be moved under the `allInOne` node above.